src/main/rx/spektrum.h

```diff
@@ -20,5 +20,5 @@
 #define SPEKTRUM_SAT_BIND_DISABLED 0
 #define SPEKTRUM_SAT_BIND_MAX 10
 
-uint8_t spektrumFrameStatus(void);
```

Comment:
When I first used CleanFlight it was frustrating figuring out how to get the spektrum working, however no matter what remote you are using, you really should change the range of your endpoints.   I have multiple Spektrum radios, and all the endpoints are not the same.   So the end point part of this modification may fix it for the DX6i, however it will break it for the DX9 or the DXE for example.   It also seems that going from an endpoint of 147, to 125 would actually lower the resolution of the signal you are receiving.   I am all for making this easier to get going on cleanflight, I just think a change in the configurator would fix this for ALL radios, rather then changing it in the firmware code.